src/rgw/rgw_sal_motr.cc

```diff
@@ -2739,10 +2753,20 @@ int MotrMultipartUpload::init(const DoutPrefixProvider *dpp, optional_yield y,
   if (rc < 0)
     return rc;
   string tenant_bkt_name = get_bucket_name(bucket->get_tenant(), bucket->get_name());
+  string upload_id = get_upload_id();
+
+  if(upload_id.length()== 0){
+    rc = store->get_upload_id(tenant_bkt_name, mp_obj.get_key(), upload_id);
+    if (rc < 0) {
+      ldpp_dout(dpp, 0) << "ERROR: get_upload_id failed. " << rc << dendl;
+      return rc;
+    }
+  }
 
   // Create object part index.
   // TODO: add bucket as part of the name.
```

Comment:
Is this TODO comment still valid ? If not remove it

Reply:
Removed